src/askem_beaker/contexts/mira/procedures/python3/visualize_mira_models.py

```diff
@@ -0,0 +1,26 @@
+import base64
+
+from mira.metamodel import TemplateModel
+from mira.modeling.viz import GraphicalModel
+
+
+def __visualize_model(name: str, model: TemplateModel) -> str:
+    graphical_model = GraphicalModel.from_template_model(model)
+    filename = f"_preview_{name}.png"
+    graphical_model.write(filename, format="png")
+    with open(filename, "rb") as f:
+        data = f.read()
+        enc = base64.b64encode(bytes(data))
+        return enc.decode("utf-8")
```

Comment:
This will actually leave files in the "jupyter" user's home directory, since subkernels currently actually run as a normal kernel in the same environment. We probably don't want to create clutter.
We should probably delete these files after we read them, or write to a IOBuffer so the contents are all in memory.